contracts/polygon/connectors/fluidity/helper.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.8.6;
+pragma experimental ABIEncoderV2;
+
+import { DSMath } from "../../common/math.sol";
+import { Basic } from "../../common/basic.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./interface.sol";
+
+abstract contract Helpers is DSMath, Basic {
+	P1M2 internal constant p1m2 =
```

Comment:
```suggestion
	IProtocolMoudule internal constant protocolModule =
```
Make changes in the rest of the code accordingly

Reply:
done